plugins/shared-react/src/utils/pipeline/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './pipeline';
+export * from './task-run';
```

Comment:
I removed the `-utils` suffix as it is redundant because its under the `utils` folder.

Reply:
now probably we can do this change but once it's published then it may break plugins(topology / tetkon) that are consuming it, once they update `shared-react` package

cc @debsmita1 / @divyanshiGupta ^^

Reply:
FYI @debsmita1 / @divyanshiGupta